tools/csv-merger/csv-merger.go

```diff
@@ -89,6 +94,7 @@ var (
 		"Comma separated list of all the CRDs that should be visible in OLM console")
 	relatedImagesList = flag.String("related-images-list", "",
 		"Comma separated list of all the images referred in the CSV (just the image pull URLs or eventually a set of 'image|name' collations)")
+	crdDir              = flag.String("crds-dir", "", "the directory containing the CRDs for apigroup validation")
```

Comment:
please modify the help message saying that the validation will be performed only if the value is != ""

Reply:
Done, and I removed the triggering of the test from `build-manifests.sh`